modules/update-duration-cvo.adoc

```diff
@@ -12,3 +12,27 @@ The Cluster Version Operator (CVO) retrieves the target update release image and
 ====
 The CVO phase of the update does not restart the nodes.
 ====
+
+[discrete]
+[id="update-duration-example_{context}"]
+== Example update duration of cluster Operators
+
+// image to be replaced with a properly designed version
+image::update-duration.png[A diagram displaying the periods during which cluster Operators report as progressing during an update]
+
+The previous diagram shows an example of the time that cluster Operators might take to update to their new versions.
+The bars on the graph represent periods during which the respective Operator reports a `Progressing=True` condition.
+During an update, the `Progressing=True` condition is triggered mostly when the Operator is updating itself, but for some Operators, other tasks related to the update may trigger the condition as well.
```

Comment:
This statement is creating the impression that operators are not  consistent and `Progressing=True` is little vague about what it represents.  Do we need to write about progressing=true here? More information about it is in https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/dev/clusteroperator.md